tooling/greybox_fuzzer/src/lib.rs

```diff
@@ -1023,6 +995,14 @@ impl<
         // Parse results
         match (result_acir, result_brillig) {
             (Ok(witnesses), Ok((_map, brillig_coverage))) => {
+                // If we expect the program to always fail, we need to return a counter example
```

Comment:
I just noticed that this block is duplicated in two more places. Well, not exactly duplicated but it's almost exactly like the other two cases. I wonder if a function couldn't be extracted where `coverage` can be passed as an `Option`, together with the durations (passing 0 when we don't have one, like you are doing now).

Reply:
Moved it out